ASP.NET Core/PdfViewerWebService_3.0/Controllers/PdfViewerController.cs

```diff
@@ -154,8 +154,8 @@ public PdfViewerController(IHostingEnvironment hostingEnvironment, IMemoryCache
         [HttpPost("ImportAnnotations")]
         [Microsoft.AspNetCore.Cors.EnableCors("MyPolicy")]
         [Route("[controller]/ImportAnnotations")]
-        //Post action to import annotations
-        public IActionResult ImportAnnotations([FromBody] Dictionary<string, string> jsonObject)
+        //Post action to import annotations
```

Comment:
Remove the .Net3.0 sample.

Reply:
I have removed the .net 3.0 sample and updated the PR